app/controllers/static_pages_controller.rb

```diff
@@ -1,3 +1,5 @@
-#home controller for first page 
+#home controller for first page
 class StaticPagesController < ApplicationController
-end
+	#if user is login than the micropost create
```

Comment:
ForI didn't get it...can you explain why you have added this comment?

Reply:
write some code for micropost but after remove them but forget the removing comments